src/vs/workbench/browser/workbench.contribution.ts

```diff
@@ -154,7 +154,7 @@ const registry = Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Con
 				'type': 'number',
 				'default': 160,
 				'minimum': 50,
-				'markdownDescription': localize('workbench.editor.tabSizingFixedMaxWidth', "Controls the maximum width of tabs when {0} is set to {1}.", '`#workbench.editor.tabSizing#`', '`fixed`')
+				'markdownDescription': localize({ comment: ['This is the description for a setting. Values surrounded by single quotes are not to be translated.'], key: 'workbench.editor.tabSizingFixedMaxWidth' }, "Controls the maximum width of tabs when `#workbench.editor.tabSizing#` size is set to `fixed`.")
```

Comment:
Translators tend get this wrong (even if there is a clear message)
@TylerLeonhardt suggested to have a variable that includes the `#`:
https://github.com/microsoft/vscode/blob/743911e92b5543ccb47c9e63d9f9cb1b67f56dcd/src/vs/workbench/services/themes/common/themeConfiguration.ts#L171-L172

Reply:
I am fine doing a change but then we should do it really in all places and do some linting to warn if you miss to use it.